### DISCUSSION TITLE
How to Translate Ai response to another language  ?[..with google translate]

### DISCUSSION THREAD

**Discussion by AloneWX** [Zain-ul-din/whatsapp-ai-bot#32] [Q&A] [answered]
I have an idea.To translate the message received by whatsapp bot in whatever language it is in English and to present the response from Ai in other language through Bot.. Please I need help.

**Comment by Zain-ul-din** [comment, marked as answer]
@Akiladinuka you have to do it manually in code using `google` or some translator APIS. You can start with [this code](https://github.com/Zain-ul-din/whatsapp-ai-bot/blob/master/src/lib/WhatsAppClient.ts#L65).

Apart from that, I think you can ask an AI model to translate your message. have you tried this yet?

**Reply by AloneWX** [reply]
@Zain-ul-din  

Thanks for helping.
